config-templates/ibm/ibm-object-storage-plugin/2.2/custom-parameters.json

```diff
@@ -0,0 +1,42 @@
+[
+    {
+        "description": "Release name of the chart",
+        "displayname": "Helm Chart Release Name",
+        "name": "helm-release-name",
+        "category": "config",
+        "required": "true"
+    },
```

Comment:
Here let us set `"required": "false"`
and `"default": "ibm-object-storage-plugin"`